$OpenBSD: patch-Bio_Tools_SiRNA_Ruleset_tuschl_pm,v 1.1 2019/02/21 10:25:48 cwen Exp $
Avoid some of the super annoying warnings about deprecated braces
Index: Bio/Tools/SiRNA/Ruleset/tuschl.pm
--- Bio/Tools/SiRNA/Ruleset/tuschl.pm.orig
+++ Bio/Tools/SiRNA/Ruleset/tuschl.pm
@@ -178,8 +178,8 @@ sub _get_oligos {
 	    my $target = $2;
 
 	    # check for too many Gs (or Cs on the other strand)
-	    next if ( $target =~ /G{ $self->gstring,}/io );
-	    next if ( $target =~ /C{ $self->gstring,}/io );
+	    next if ( $target =~ /G\{ $self->gstring,\}/io );
+	    next if ( $target =~ /C\{ $self->gstring,\}/io );
 # 	skip Ns (for filtering)
 	    next if ( $target =~ /N/i);
 
